centreon-open-tickets/www/modules/centreon-open-tickets/providers/Abstract/AbstractProvider.class.php

```
@@ -933,8 +937,26 @@ protected function getConfigContainer1Main()
+            . '/><label class="empty-label" for="peer_verify"></label></div>';
+        $caCertPathHtml  = '<input size="50" name="ca_cert_path" type="text" value="' . $this->getFormValue('ca_cert_path') . '" />';
+
+        $array_form['peer_verify'] = ['label' => _('SSL Verify Peer'), 'html' => $peerVerifyHtml];
+        $array_form['ca_cert_path'] = ['label' => _('Certificate Authority Info'), 'html' => $caCertPathHtml];
 
         $tpl->assign('form', $array_form);
+
+        // prepare help texts
+        $helptext = '';
+        include_once 'help.php';
+        foreach ($help as $key => $text) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$help'. (undefined)

(UnusedLocalVariable)